### PR TITLE
fix(cli): retain stopped subworkflows in task picker

### LINKED ISSUE #4879: CLI task picker hides stopped sub-workflows
## Summary

The CLI TUI retains stopped child streams in the stream strip and subagent picker, but the `Tasks and sub-workflows` picker drops stopped sub-workflows after they are killed. That means users cannot reopen the task detail view for a stopped sub-workflow to inspect its final output.

## Repro

From current `origin/main` at `1b5cbee6a`:

```sh
corepack pnpm install
corepack pnpm --filter @texra-ai/cli bundle:harness
TEXRA_TUI_HARNESS=$PWD/packages/cli/dist/bin/tui-harness.js \
  node packages/cli/scripts/validate-tui.mjs \
  --snapshot-dir /tmp/texra-stopped-task-current stopped-subagent-picker focused-stopped-subagent
```

For a direct manual check in the harness: open `Option-p`, press `k` on the strategy sub-workflow, then open `Option-p` again.

## Observed

After killing `strategy`, the task picker shows only remaining active work:

```text
Tasks and sub-workflows
Stream: main
› 1. leanSolver — waiting ...
  2. reviewer — running ...
  3. latex build — running ...
```

The stream strip still shows `3:strategy(stopped)`, and the subagent picker still retains the stopped strategy row.

## Expected

`Tasks and sub-workflows` should include retained stopped/completed child streams, so users can select the stopped sub-workflow and inspect its detail/output.

## Likely cause

`buildChildControlItems(..., 'tasks')` uses only `slice.activeSubagents` plus `slice.activeProcesses`, while subagent mode uses `visibleSubagentRows(slice)` to include retained child streams.

### PR DESCRIPTION
**Pull request by LionSR**
## Summary

- include retained child streams in the `Tasks and sub-workflows` picker, matching the subagent picker and stream strip
- keep stopped/completed retained sub-workflow rows non-killable so their picker/detail actions do not advertise stale `k kill`
- add kernel and PTY coverage for stopped task picker rows and stopped sub-workflow detail inspection

Fixes #4879.

## Validation

- `npm run format -- --log-level warn`
- `npm run test:kernel -- src/test-kernel/cli/ChildControls.vitest.mts`
- `corepack pnpm --filter @texra-ai/cli bundle:harness`
- `TEXRA_TUI_HARNESS=$PWD/packages/cli/dist/bin/tui-harness.js node packages/cli/scripts/validate-tui.mjs --snapshot-dir /tmp/texra-stopped-task-fixed stopped-task-picker stopped-task-subworkflow-detail stopped-subagent-picker focused-stopped-subagent task-picker task-subworkflow-detail`
- `TEXRA_TUI_HARNESS=$PWD/packages/cli/dist/bin/tui-harness.js node packages/cli/scripts/validate-tui.mjs --snapshot-dir /tmp/texra-stopped-task-full`
- `npm run compile:safe`
- `npm run lint` (0 errors; existing 12 import-order warnings)
- `git diff --check`

---

> [!NOTE]
> **Low Risk**
> Localized TUI child-control state and test-only harness scenarios; no auth, persistence, or network surface.
> 
> **Overview**
> The **Tasks and sub-workflows** picker now lists **retained** sub-workflow rows from `childStreams` (via `visibleSubagentRows`), not only currently active subagents—aligned with the subagent picker and stream strip.
> 
> Stopped or completed retained rows are **`killable: false`**, so picker and task-detail UIs no longer show **`k kill`** for work that is already finished. Task visibility checks (`hasVisibleTasks`) use the same retained-row model.
> 
> **Tests:** kernel expectations for mixed active/stopped rows; parent fallback when only stopped child streams remain; PTY harness scenarios **`stopped-task-picker`** and **`stopped-task-subworkflow-detail`**.
> 
> <sup>Reviewed by [Cursor Bugbot](https://cursor.com/bugbot) for commit fbeaaf215219a9d8c5acbfaba2991251e2a3bffb. Bugbot is set up for automated code reviews on this repo. Configure [here](https://www.cursor.com/dashboard/bugbot).</sup>

### CODE REVIEW

File: packages/cli/src/chat/tui/state/childControls.ts [hunk 1]
```diff
@@ -214,9 +214,15 @@ export function buildChildControlItems(
...
   return [
-    ...slice.activeSubagents.map((child) =>
-      buildSubagentItem(child, streamsById, nowMs),
+    ...visibleSubagentRows(slice).map((child) =>
+      buildSubagentItem(
+        child,
+        streamsById,
+        nowMs,
+        activeKeys.has(childKey(child)),
+      ),
```
[the remark this answers is not in the document]
Comment (LionSR) [reply packages/cli/src/chat/tui/state/childControls.ts:225]: Fixed in the updated head `fbeaaf215`: `hasVisibleTasks` now uses `visibleSubagentRows(slice)` and its slice type includes `childStreams`, matching the task picker row builder and `hasChildExecutionRows`. I also added a unit test for the exact edge case: a focused leaf falling back to a parent whose only task rows are retained stopped child streams. Local validation after the fix: `npm run test:kernel -- src/test-kernel/cli/ChildControls.vitest.mts`, focused PTY scenarios including `task-picker-parent-fallback`, `npm run compile:safe`, `npm run lint`, `git diff --check`, and full `validate-tui` all 67 scenarios.
